examples/text.py

```diff
@@ -1,17 +1,17 @@
 import blend2d
 import numpy as np
-from skimage.io import imsave
+import matplotlib.pyplot as plt
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     array = np.empty((500, 500, 4), dtype=np.uint8)
     image = blend2d.Image(array)
     canvas = blend2d.Context(image)
 
-    canvas.clear()
-    canvas.set_fill_style((1.0, 1.0, 1.0))
+    canvas.clear_all()
+    # canvas.set_fill_style((1.0, 1.0, 1.0))
 
-    text = u'Hello World!'
-    font = blend2d.Font('/Library/Fonts/Skia.ttf', 50)
-    canvas.fill_text((100, 100), font, text)
-
-    imsave('text.png', array)
+    text = "Hello World!"
+    # font = blend2d.Font("/Library/Fonts/Skia.ttf", 50)
+    # canvas.fill_text((100, 100), font, text)
```

Comment:
I'm guessing this slipped in? Removing this code turns this example into a no-op.

Reply:
You are right, this slipped in. However , the font does not exist in my system and so it did not really worked. I will go back to the initial example.